ChainOfResponsibility/src/RequestHandlerA.java

```diff
@@ -0,0 +1,12 @@
+
+public class RequestHandlerA extends RequestHandler {
+
+	public void handleRequest(Request request) {
+		if (request.getValue() < 0) {
+			System.out.println(request.getDescription());
+		}
+		else {
+			super.nextRequestHandler.handleRequest(request);
```

Comment:
If this request handler can't handle the request, it passes it to the next request handler.